src/keboola_mcp_server/tools/storage.py

```diff
@@ -103,6 +103,9 @@ class TableColumnInfo(BaseModel):
         validation_alias=AliasChoices('quotedName', 'quoted_name', 'quoted-name'),
         serialization_alias='quotedName',
     )
+    data_type: Optional[str] = Field(None, description='Data type of the column.')
+    base_type: Optional[str] = Field(None, description='Base data type of the column.')
+    nullable: Optional[bool] = Field(None, description='Whether the column can contain null values.')
```

Comment:
I'm not a big fan of `Optional[bool]` because now you have a third state you have to handle. Perhaps this should be `True` by default.